KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/KeywordSearchIngestModule.java

```diff
@@ -547,17 +591,17 @@ public Reader openStream() throws IOException {
                 }
                 //divide into chunks and index
                 return Ingester.getDefault().indexText(finalReader, aFile.getId(), aFile.getName(), aFile, context);
-            } catch (TextExtractorFactory.NoTextExtractorFound | TextExtractor.InitReaderException ex) {
+            } catch (TextExtractor.InitReaderException ex) {
```

Comment:
I think the comment that follows here needs to be relocated. We did find a text extractor.